src/components/IOUConfirmationList.js

```diff
@@ -258,6 +313,39 @@ class IOUConfirmationList extends Component {
         ];
     }
 
+    /**
+     * Adds Venmo, if available, as the second option in the menu of payment options
+     */
+    addVenmoPaymentOption() {
```

Comment:
This way the venmo option will always appear last right? Since the component should already be mounted with the other options or is that not the case?

Reply:
> This way the venmo option will always appear last right?

I adjusted how we put the Venmo option into the array in the `setState` call below so that it would always appear second